sys-libs/libseccomp/libseccomp-2.5.5-r1.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{9..11} )
+DISTUTILS_OPTIONAL=1
+
+inherit distutils-r1 multilib-minimal
+
+DESCRIPTION="high level interface to Linux seccomp filter"
+HOMEPAGE="https://github.com/seccomp/libseccomp"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/seccomp/libseccomp.git"
+	PRERELEASE="2.6.0"
+	inherit autotools git-r3
+else
+	SRC_URI="https://github.com/seccomp/libseccomp/releases/download/v${PV}/${P}.tar.gz
+		experimental-loong? ( https://dev.gentoo.org/~xen0n/distfiles/${PN}-2.5.5-loongarch64-20231204.patch.xz )"
+	# NOTE: restore on next revbump
+	# This rev is only for bumping the experimental-loong patchset, so ~loong-only.
```

Comment:
Is 2.5.5 broken on loong at runtime or just build-time? If the latter, I'd prefer we just edit 2.5.5 to avoid the revision mess.

Reply:
Hmmm, just build-time. I've updated to just amend 2.5.5's `SRC_URI`.